### ISSUE TITLE
Facebook Messenger dark mode have white bubbles

### ISSUE THREAD

**Issue by dominikmartini** [ferdium/ferdium-app#1426]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.5.2

### What Operating System are you using?

macOS

### Operating System Version

Sonoma 14.1

### What arch are you using?

arm64 (including Apple Silicon)

### Last Known Working Ferdium version

IDK

### Expected Behavior

The bubbles should be as normal in the theme of chat

### Actual Behavior

Bubbles from the other side of chat (not msgs from me) should be in theme color

### Steps to reproduce

I just look at FB MSG chat and there it is :D 

### Screenshots

<img width="1511" alt="Snímek obrazovky 2023-10-26 v 23 18 32" src="https://github.com/ferdium/ferdium-app/assets/143851830/0c52e407-966c-4b42-b39f-d934fc612390">

**Comment by SpecialAro** [comment]
Hello

I cannot stress this enough but please do not open duplicates of issues before looking at the issue board for similar issues.

This is a duplicate of #1416
